CefSharp.WinForms/ChromiumWebBrowser.cs

```diff
@@ -93,10 +94,31 @@ protected override void Dispose(bool disposing)
             base.Dispose(disposing);
         }
 
+        void movingListener_Moving(object sender, EventArgs e)
+        {
+            if (IsBrowserInitialized)
+            {
+                NotifyMoveOrResizeStarted();
+            }
+        }
+
         void IWebBrowserInternal.OnInitialized()
         {
             IsBrowserInitialized = true;
 
+            // By the time this callback gets called, this control
+            // is most likely hooked into a parent Form of some sort. 
+            // (Which is what MovingListener relies on.)
+            // Ensure the MovingListener contruction occurs on the WinForms UI thread:
+            if (InvokeRequired)
```

Comment:
There's an extension method `InvokeOnUiThreadIfRequired`, prefer to use that for consistency/brevity.